Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM renovate/node@sha256:5f9461153755d482ebf16a5ebf57bda3c8ec9a9ad090aacf6381170ce1340d67
```

Comment:
I noticed cases when `node` binary was used to fetching

Reply:
This should be based off our latest ruby image

Reply:
Do you mean cocoapods needs `node` ??

Reply:
Not in the all cases, but [here is one example](https://github.com/l2succes/react-native-mapbox-gl/blob/master/react-native-mapbox-gl.podspec#L17).

Reply:
We should try to skip such scripts if possible